src/helpers.coffee

```diff
@@ -31,13 +32,16 @@ jade = async ({source, target, data}) ->
   render = _jade.compile source.content, filename: source.path
   target.content = render data
 
+handlebars = async ({source, target, data}) ->
+  source.content ?= yield read source.path
+  target.content = _handlebars source.content, data
```

Comment:
Hmmm... Looks like `sass` and `stylus` use promises, but `coffee` and `jade` do not. Looking at the `jade` implementation of `compile`, it is just returning a function, doesn't seem to be promise aware at all, unless I'm missing something. It looks like both `sass` and `stylus` have a callback function associated with compilation, but that's not true of `handlebars`. So my guess is that those need to be wrapped in promises so the next yield waits until they resolve, but handlebars (and jade and coffee) compilation are synchronous, and so no promises is needed.

@dyoder Can you confirm? Thanks!